### PR TITLE
Add common paragraphs

### PR DESCRIPTION
**Pull request by bboro**
#273 [8h]

<a href="https://gitpod.io/#https://github.com/Gizra/drupal-starter/pull/285"></a>

### CODE REVIEW

**Comment by bboro** [comment]
Admin form:

https://user-images.githubusercontent.com/24897663/186914089-4f0dee7d-2124-4461-a55f-4197b4c109b5.mp4

Styleguide:

https://user-images.githubusercontent.com/24897663/186914107-4402ff1f-eef7-43da-b910-0ddde7e8ebb4.mp4

Wire-up
375|640|768|1440
-|-|-|-
Screen Shot 2022-08-26 at 14 27 36|Screen Shot 2022-08-26 at 14 27 41|Screen Shot 2022-08-26 at 14 27 46|Screen Shot 2022-08-26 at 14 27 51

**Comment by bboro** [comment]
TB: 3 / 8h

**Comment by bboro** [comment]
- Added related content paragraph
- Added carousel-base twig
- Added carousel js
- Added styleguide example
- Wire up related content paragraph

https://user-images.githubusercontent.com/24897663/186954565-ff140fad-e3e9-4d86-aef9-de8eb3299497.mp4

TB: 5.5 / 8h

File: web/themes/custom/server_theme/src/js/slick-carousel.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        }
+        else if (isResponsive) {
+          slidesToShow = 1;
+        }
+        else {
+          slidesToShow = numSlides > 3 ? 3 : numSlides;
+        }
+
+        const config = {
+          infinite: TRUE,
```
Comment (amitaibu) [inline web/themes/custom/server_theme/src/js/slick-carousel.js:91]: TRUE and FALSE won't work. Going to replace

**Comment by amitaibu** [comment]
- Make `Is title hidden` on Landing page work
- Remove phpcs from JS, as it changes `true` to `TRUE`
- Migrate content into the `Related content` paragraph type, and add it to the Homepage landing page
- Fix button to be `inline-block`, so it takes up space correctly
- Re-add `pcss` decelerations to Slick's dots.

| Mobile | Tablet |
|-----|---------|
| image | image |

**Pull request merged**